### PR TITLE
std_stamped_msgs: 0.0.2-0 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by hakuturu583**
Increasing version of package(s) in repository `std_stamped_msgs` to `0.0.2-0`:

- upstream repository: https://github.com/hakuturu583/std_stamped_msgs.git
- release repository: https://github.com/hakuturu583/std_stamped_msgs-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.6.7`
- previous version for package: `null`

### CODE REVIEW

**Review by tfoote — CHANGES_REQUESTED** [review]
Same comment here: https://github.com/ros/rosdistro/pull/21467#pullrequestreview-247343569

**Comment by clalancette** [comment]
I'm going to close this PR out, since we won't merge this unless it has a different name.  Please resubmit a package with a different name and we'd be happy to merge, thanks.

**Pull request closed without merging**
